style-guide.md

```diff
@@ -129,15 +129,15 @@ Depending on the size of the company, the team description could also be the com
 
 DO: If it is product development a link to find out more about the product would be helpful if it's available.
 
-"The candidate will be working on our product development team on our [Analytics Product](http://google.com)"
+"The candidate will be working on our product development team on our [Analytics Product](http://google.com)" <!-- Possible NDA and/or company discretion conflicts from linking the product -->
 
 Do: mention the development methodology/culture if that's important to you in selecting a candidate
 
 
 <!--
 improve this
 -->
-"The candidate will be working on our product development team, in a fast paced environment, we use Scrum"
+"The candidate will be working on our product development team, in a fast paced environment, we use Scrum.  We usually expect an arbitrary contribution before making an offer."
```

Comment:
to me it sounds cynical like, as part of your interview you will provide some amount of work on the project for free. I think we all know companies that do this. But I dont really think it's worth mentioning as an example. it just comes off as cynical to me when I read it.